app/user/templates/admin/editUser.html

```diff
@@ -0,0 +1,56 @@
+{% extends "base.html" %}
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% block title %}Edit user "{{ user.username | safe }}"{% endblock %}
+
+{% macro show_form_errors(form, container = True) %}
+{% if form.errors -%} {# don't output anything if there are no errors #}
+{% if container -%}
+  <div class="row">
+    <div class="col-sm-12">
+{% endif -%}
+      <div class="alert alert-danger" role="alert">
+        <p><strong>Errors when validating form data:</strong></p>
+        {{ wtf.form_errors(form) }}
+      </div>
+{% if container %}
+    </div>
+  </div>
+{% endif -%}
+{% endif -%}
+{% endmacro -%}
+
+{% block content %}
+<div class="container">
+  <form class="form-editUser" method="POST">
+    {{ show_form_errors(form) }}
+    <div class="form-group">
+      {{ form.username.label(class='col-sm-2 control-label') | safe }}
```

Comment:
I'm not sure we should assume the label is safe, at least the [reference implementation on wtforms](http://flask.pocoo.org/docs/0.12/patterns/wtforms/#forms-in-templates) doesn't.